### ISSUE TITLE
`theme-color` always equals to #FFFFFF when RFG used for icon generation

### ISSUE THREAD

**Issue by Iwasawafag** [haydenbleasel/favicons#127]
When RealFaviconGenerator is being used, regardless of `background` value passed in configuration object in generated HTML it always says:

```
<meta name="theme-color" content="#FFFFFF">
```

~~I also tried to modify default value (`favicons.config.rfg.favicon_design.android_chrome.theme_color`) but that changed nothing.~~ 
**UPD:** Actually, no, changing it manually does the trick

**Comment by Iwasawafag** [comment]
I see why. Current code [mistakendly assumes](https://github.com/haydenbleasel/favicons/blob/04cee02/helpers.js#L266) that `theme_color` specidied under `manifest` object 

```js
request.favicon_design.android_chrome.manifest.theme_color = options.background;
```

when should be 

```js
request.favicon_design.android_chrome.theme_color = options.background;
```

**Comment by haydenbleasel** [comment]
Sorry for the delay @Iwasawafag, fixed in `favicons@4.7.5`.
